x-pack/plugins/upgrade_assistant/public/application/components/overview/review_logs_step/no_deprecations/_no_deprecations.scss

```diff
@@ -1,3 +0,0 @@
-.upgRenderSuccessMessage {
```

Comment:
@sabarasaba I removed this style, as I think the same layout can be accomplished with [`EuiSpacer`](https://elastic.github.io/eui/#/layout/spacer).